node-puppeteer-chrome/puppeteer_download.js

```diff
@@ -15,7 +15,14 @@ async function downloadLatestCompatibleChrome() {
     // Get all supported Chrome version
     const compatibleChromeVersion = matchedCompatibilityVersions
         .map((v) => v.chrome)
-        .sort((a, b) => b.localeCompare(a));
+        .sort((a, b) => {
+            const [majorA, minorA] = a.split('.').map((v) => Number(v));
+            const [majorB, minorB] = b.split('.').map((v) => Number(v));
+
+            if (majorB > majorA) return 1;
+            if (minorB > minorA) return 1;
+            return b.localeCompare(a);
```

Comment:
isnt this wrong? i mean if you compare a=3.2.1 vs b=3.2.10, both conditions are false and you end up doing local compare which will fail to sort this, or not?

might be safer to build a number from the version string (e.g. patch + 100 * minor + 10000 * major) and compare those